pom.xml

```diff
@@ -64,11 +64,14 @@ under the License.
 
   <properties>
     <javaVersion>8</javaVersion>
+    <aetherVersion>1.0.0.v20140518</aetherVersion>
     <mavenVersion>3.2.5</mavenVersion>
     <checkstyleVersion>9.3</checkstyleVersion>
     <doxiaVersion>1.11.1</doxiaVersion>
-    <sitePluginVersion>3.9.1</sitePluginVersion>
-    <surefire.version>2.22.2</surefire.version>
+    <doxiaSitetoolsVersion>1.11.1</doxiaSitetoolsVersion>
+    <sitePluginVersion>3.12.1</sitePluginVersion>
+    <jxrPluginVersion>3.3.0</jxrPluginVersion>
+    <surefire.version>3.0.0-M7</surefire.version>
```

Comment:
`surefire.version` the same value in parent 37

Reply:
Done